wurst/systems/entities/bosses/AlligatorMan.wurst

```diff
@@ -2,7 +2,30 @@ package AlligatorMan
 import ID
 import Hostile
 import LinkedList
+import BossUtilities
```

Comment:
Please separate out imports. It makes it easier to visualize what we have control over.